deepchem/models/torch_models/dtnn.py

```diff
@@ -0,0 +1,150 @@
+from typing import List
+
+import torch.nn as nn
+import torch
+
+from deepchem.models.torch_models import layers
+
+
+class DTNN(nn.Module):
```

Comment:
This should be listed in docs/. Let's just add it to layers.rst.

Reply:
Done